docs/conf.py

```diff
@@ -45,7 +45,7 @@
 
 # General information about the project.
 project = 'django-crispy-forms'
-copyright = '2009-2013, Miguel Araujo and Daniel Greenfeld'
+copyright = '2009-2021, Miguel Araujo, Daniel Feldroy and contributors'
```

Comment:
Since it's Python code we could have `datetime.now().year` . I've left it to be changed manually for now (so we change the LICENSE as well, etc.).

The LICENSE already had "contributors" but not Daniel - all seems good now.